pytest_fixtures/component/provision_pxe.py

```diff
@@ -245,8 +250,23 @@ def provisioning_host(module_ssh_key_file, pxe_loader, module_provisioning_sat):
     ) as prov_host:
         yield prov_host
         # Set host as non-blank to run teardown of the host
-        assert module_provisioning_sat.sat.execute('systemctl restart dhcpd').status == 0
-        prov_host.blank = getattr(prov_host, 'blank', False)
+        if not settings.server.is_ipv6:
+            assert module_provisioning_sat.sat.execute('systemctl restart dhcpd').status == 0
+            prov_host.blank = getattr(prov_host, 'blank', False)
+
+
+@pytest.fixture(scope='module')
+def configure_kea_dhcp6_server():
+    if settings.server.is_ipv6:
+        kea_host = Broker(
+            workflow='configure-kea-dhcp6',
+            artifacts='last',
+            host_class=ContentHost,
+            blank=True,
+            target_vlan_id=settings.provisioning.vlan_id,
+        ).execute()
+        yield kea_host
+        Broker(workflow='remove-vm', source_vm=kea_host['_broker_facts']['name'][0]).execute()
```

Comment:
Does `kea_host` contains  _broker_facts here, and is checkin working correctly here, could you confirm with ContentHost object?

Reply:
My bad, it worked with previous implementation. Updated to correct one now